.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: krew-index CI
+
+on:
+- push
+- pull_request
+
+jobs:
+  build:
+    name: Build and Tags
+    runs-on: ubuntu-latest
+    env: 
+      KREW_VERSION: v0.3.3
+      GO111MODULE: on
+    steps:
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0  # disable shallow clones
+
+    - name: Install kubectl
+      run: |
+        set -euo pipefail
+        sudo curl -fsSL -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.13.1/bin/linux/amd64/kubectl
+        sudo chmod +x /usr/bin/kubectl        
+
+    - name: Install krew
+      run: |
+        set -euo pipefail
+        cd "$(mktemp -d)"
+        curl -fsSLO "https://github.com/kubernetes-sigs/krew/releases/download/${KREW_VERSION}/krew.{tar.gz,yaml}"
+        tar zxvf krew.tar.gz
+        ./krew-linux_amd64 install --manifest=krew.yaml --archive=krew.tar.gz
+        
+    - name: Install validate-krew-manifest plugin
+      run : |
+        set -euo pipefail
+        export GOBIN=$HOME/bin
+        go get sigs.k8s.io/krew/cmd/validate-krew-manifest@${KREW_VERSION}
```

Comment:
I'm just wondering why this even works. In the manifest for `krew` GH actions, we had to setup go. But not here, even though this clearly needs go.

Do you know what's going on here? @alonyb

Reply:
@corneliusweig 
Yes, `ubuntu-latest` contains some versions of Go. You can see that [here](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/software-installed-on-github-hosted-runners#ubuntu-1804-lts), but anyway I will put go 1.13 in another step to be clear.